template/src/services/networking/axios.ts

```diff
@@ -0,0 +1,106 @@
+import {AnyAction, Store} from '@reduxjs/toolkit'
+import axios, {AxiosError, AxiosRequestConfig, AxiosResponse} from 'axios'
+import Config from 'react-native-config'
+import {AXIOS_TIMEOUT, RESPONSE_CODE, TOKEN, TOKEN_TYPE} from '../../constants'
+import {userActions} from '../../store/reducers'
+import {RootState} from '../../store/store'
+import {getData, setData, clearAllData} from '../../utilities/storage'
+import {AUTH_API} from '../api/api'
+
+let store: Store<RootState, AnyAction>
+
+const instance = axios.create({
+  baseURL: Config.API_URL,
+  timeout: AXIOS_TIMEOUT,
```

Comment:
This is not like what written in the docs above

Reply:
done